console/client/src/features/modules/ModulesRequests.tsx

```diff
@@ -0,0 +1,124 @@
+import React from 'react'
+import { useSearchParams } from 'react-router-dom'
+import { Panel } from './components'
+import { modulesFilter } from '../../services/console.service'
+import { streamEvents } from '../../services/console.service'
+import { Event, Module, EventsQuery_Filter } from '../../protos/xyz/block/ftl/v1/console/console_pb'
+import { TimelineCall } from '../timeline/TimelineCall.tsx'
+import { TimelineDeploymentCreated } from '../timeline/TimelineDeploymentCreated.tsx'
+import { TimelineDeploymentUpdated } from '../timeline/TimelineDeploymentUpdated.tsx'
+import { TimelineIcon } from '../timeline/TimelineIcon.tsx'
+import { TimelineLog } from '../timeline/TimelineLog.tsx'
+import { TimelineCallDetails } from '../timeline/details/TimelineCallDetails.tsx'
+import { TimelineDeploymentCreatedDetails } from '../timeline/details/TimelineDeploymentCreatedDetails.tsx'
+import { TimelineDeploymentUpdatedDetails } from '../timeline/details/TimelineDeploymentUpdatedDetails.tsx'
+import { TimelineLogDetails } from '../timeline/details/TimelineLogDetails.tsx'
+import { SidePanelContext } from '../../providers/side-panel-provider.tsx'
+import { formatTimestampShort } from '../../utils/date.utils.ts'
+import { panelColor } from '../../utils/style.utils.ts'
+import { Timestamp } from '@bufbuild/protobuf'
+
+const maxTimelineEntries = 1000
+
+export const ModulesRequests: React.FC<{
+  className?: string
+  modules: Module[]
+}> = ({ className, modules }) => {
+  const [searchParams, setSearchParams] = useSearchParams()
+  const [entries, setEntries] = React.useState<Event[]>([])
+  const { openPanel, closePanel } = React.useContext(SidePanelContext)
+  const [selectedEntry, setSelectedEntry] = React.useState<Event | null>(null)
+  const deployments = modules.map(({ deploymentName }) => deploymentName)
+
+  const filters: EventsQuery_Filter[] = []
+  if (deployments.length) {
+    filters.push(modulesFilter(deployments))
+  }
```

Comment:
This component is named `ModuleRequests`, so I just want to point out this is getting all events for a module. This would likely include `Logs`, `Calls`, `Deployments` etc. Is that what you're looking for here?

Reply:
For the moment yes. I need to site down and and figure out how to move to a more request trace based log like in the wireframes. Talk about it tomorrow?